README.md

```diff
@@ -296,6 +296,12 @@ Cell-level layouts:
 - Bounding box
 - Area usage in nm²
 
+## Benchmark Library
+
+To objectively evaluate and compare software and design automation tools, [MNT Bench](https://www.cda.cit.tum.de/mntbench/) provides gate-level
+layouts for various gate libraries and clocking schemes, generated using the latest physical design and
+optimization algorithms, with *fiction* offering the correpsonding read and write utitlities to generate gate-level layouts from gate-level layout files (``.fgl``) and vice versa.
```

Comment:
```suggestion
## Benchmark Library

To objectively evaluate and compare software and design automation tools, [MNT Bench](https://www.cda.cit.tum.de/mntbench/) provides gate-level
layouts for various gate libraries and clocking schemes, generated using the latest physical design and
optimization algorithms, with *fiction* offering the corresponding read and write utilities to generate gate-level layouts from gate-level layout files (``.fgl``) and vice versa.
```
So far, this doesn't discuss the Verilog files available in MNT Bench. Do you want to mention that still?